cmd/thanos/receive.go

```diff
@@ -488,7 +494,7 @@ func setupHashring(g *run.Group,
 			defer close(updates)
 			updates <- ring
 			<-cancel
-			return nil
+			return hashringConfigs, nil
```

Comment:
I don't think this can be returned here. We're only sending updates

Reply:
We rejected this approach now. But let's clarify why we cannot return `hashringConfigs` here in the next meeting, so that I can better understand `setupHashring`!